libs/core/src/lib/select/select.component.scss

```diff
@@ -20,3 +20,9 @@
 .fd-popover-custom-select-body {
     max-width: 100%;
 }
+
+.fd-select_text-content-inShellbar {
```

Comment:
Let's not create a new class for this but instead add these styles to `fd-shellbar__input-group-input--select` via fundamental-style

Reply:
https://github.com/SAP/fundamental-styles/pull/3843